### ISSUE TITLE
Need a Load command

### ISSUE THREAD

**Issue by bayerf42** [neil-lindquist/SLIMA#29]
Currently Alt-K just compiles a buffer but doesn't load it into the Lisp process. In SLIME, both options (compile-only and compile-and-load) are available, we need this here, too.

**Comment by neil-lindquist** [comment]
Alt-k is suppose to load the buffer.  Does it fail to load even if you use alt-k on this hello world function in a fresh REPL?
``` common-lisp
(defun foobar ()
  (+ 1 1))


(defun hello-world ()
  (format t "Hello World"))
```

Also, what's your lisp implementation, OS, slime version, ect?

**Comment by bayerf42** [comment]
Just tried it and it fails:

; compiling file "F:/projects/lisp/foo.lisp" (written 17 AUG 2019 08:26:26 PM):

; wrote F:/projects/lisp/foo.fasl
; compilation finished in 0:00:00.039

CL-USER> (hello-world)
The function COMMON-LISP-USER::HELLO-WORLD is undefined.
   [Condition of type UNDEFINED-FUNCTION]

I'm using SBCL 1.4.14 on Windows 7,  Atom 1.40.0 x64 and Slime 2.24

Compiling the function with Alt-C works fine.

**Comment by neil-lindquist** [comment]
Oops, I somehow read alt-k as alt-c.  I'm getting the same result as you when I actually run alt-k.
I think I know where the mistake might be, so I'll look into it once I finish the fix for #28.
